modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -5121,6 +5121,9 @@ public CodegenParameter fromParameter(Parameter parameter, Set<String> imports)
         if (parameter.getExtensions() != null && !parameter.getExtensions().isEmpty()) {
             codegenParameter.vendorExtensions.putAll(parameter.getExtensions());
         }
+        if (parameter.getSchema() != null && parameter.getSchema().getExtensions() != null && !parameter.getSchema().getExtensions().isEmpty()) {
```

Comment:
LGTM from a spring perspective - but this will impact all of java.

Do you want to modify one of the samples so that it shows what the generated result looks like?